### DISCUSSION TITLE
Welcome to Newt_Display Discussions!

### DISCUSSION THREAD

**Discussion by darianbjohnson** [Phambili-Tech/Newt_Display#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andypiper** [comment]
Hi all - I'm Andy, and I've not one but TWO Newts! 😃 one for my desk and one for hacking!

I have [a repo fork](https://github.com/andypiper/Newt_Display) where I've created a bunch of Issues which represent some of my ideas for where the base firmware could go next, and I've made a few small tweaks that I'll send PRs with at some point; but on top of that I'm excited to get MicroPython onto my spare device and to start working on some other code there.

Thanks for opening up discussions!

**Comment by jhoskins98** [comment]
Hey - John here.  Got mine a couple of weeks ago

Would love to start messing with it but I have some basic questions
- Where do you expect the community on the Newt to be?  Seems here but would be good to put in your good intro on phambili.tech.  
- How do I download the current version 1.x and put it on my Newt?  Download fails don't know if that is an issue or not.  
- How to load up a current version of software into Arduino IDE?  And then push it?  I know that you said this is an advanced board but it is the board I now have with good working software.

As for my use case, I am looking for a device to give to my senior father to show upcoming appointments (not so good about phones, etc) from Google Calendar, messages from the kids and other information that might be pertinent.  Ultimately, a bigger form factor for tired eyes would be good.  

Thanks for building such a nice product.

**Reply by andypiper** [reply]
I had download issues in part of my house, but not in another, so I would see whether you have any different results on different wifi networks or closer to your router (just for the update download).

I’ve also so far struggled to get a build transferred reliably from the Arduino IDE and would like to see a repeatable set of instructions.

Not sure whether the progress of my issue [here](https://github.com/Phambili-Tech/Newt_Display/issues/2) will help you to figure things out!

**Reply by darianbjohnson** [reply]
@andypiper - Thanks for logging your progress on the download issues... your note will help me fix some of the problem. Re; Arduino build documentation - I'll accelerate working on that.. I didn't realize people were having problems getting the build to work. I tested it on a second "clean" PC with a new arduino install, but I can/should probably do more testing to make sure that a user can down, modify, and upload the code. Logging an issue...

**Reply by darianbjohnson** [reply]
@jhoskins98 - There is a way to share a google calendar, but it requires some authentication "hoops" to jump through. That being said, I've gotten this request 3-4 times now, so I think I need to move it earlier on the roadmap. And yes - I need to update the phambili.tech/newt website. I'll handle that this weekend... logging another issue... (that's the best way to keep me honest on tracking the feedback)

**Comment by tjfoth** [comment]
Darian,

I think we are going to need some  some of scheduler, that would schedule
wake-ups and call routines to refresh data.   The calendar  and my  MQTT
needs are all examples.

---
Tom Foth
***@***.***
***@***.***

**Comment by FotOpa** [comment]
Hi everybody. I'm using Firmware V 1.1.5 on my NewT. Loading the battery, is there any indication when it is fully charged? If not, how long should I charge it until it is assumed to be fully charged? I'm charging my battery now for a week and the charging indicator in the left upper corner of the display doesn't change.
